x-pack/legacy/plugins/rollup/public/crud_app/sections/job_create/steps/step_date_histogram.js

```diff
@@ -26,7 +26,7 @@ import {
 
 import {
   parseEsInterval,
-} from 'ui/utils/parse_es_interval';
+} from '../../../../../../../../../src/legacy/core_plugins/data/public';
```

Comment:
:-(

Reply:
=( but we should use `relative `imports now

Reply:
Yeah, I know. Was just acknowledging that it's ugly.